server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -161,7 +161,18 @@ public Invocations getInvocations() {
       NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
-      return asJsonResponseFuture(cacheManager.getCacheManagerInfo().toJson(), responseBuilder);
+      Boolean rebalancingEnabled = null;
+      try {
+      rebalancingEnabled = SecurityActions.getGlobalComponentRegistry(cacheManager).getLocalTopologyManager()
+            .isRebalancingEnabled();
+      } catch (Exception e) {
```

Comment:
`SecurityException`?

Intellij is also warning about `e` being unused, so we should either add a comment in the body or change `e` to `ignore` to suppress the warning.